app/controllers/reviews_controller.rb

```diff
@@ -0,0 +1,49 @@
+class ReviewsController < ApplicationController
+  def create
+    @landmark = Landmark.find(params[:landmark_id])
+    @review = Review.new(review_params)
+    @review.landmark = @landmark
```

Comment:
Why not merge this in the params? Would de-clutter the controller method, which is getting pretty long.